pkgs/by-name/co/cosmic-player/package.nix

```
@@ -0,0 +1,96 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, rustPlatform
+, makeBinaryWrapper
+, alsa-lib
+, cosmic-icons
+, ffmpeg
+, llvmPackages
+, just
+, pkg-config
+, libxkbcommon
+, wayland
+, xorg
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cosmic-player";
+  version = "unstable-2024-06-10";
```

```suggestion
  version = "0-unstable-2024-06-10";
```
[The `version` attribute must start with a digit](https://github.com/NixOS/nixpkgs/tree/master/pkgs#package-naming)